modules/ROOT/pages/apply-oauth-token-policy-mule3.adoc

```diff
@@ -0,0 +1,34 @@
+= OAuth 2.0 Token Validation
```

Comment:
Since we are adding a whole new section for Mule 3, can we place all the new pages in a subfolder, so site is more maintainable? In the mid term, it would be great to organize all information in different subfolders (folders as buckets approach), to reduce maintenance complexity.

Reply:
Each subfolder would be an extra path in the URL. 
It would be something like docs.mulesoft.com/api-manager/policies/mule-3/...

We can research how we would add internal subfolders to organize the content a little bit better.

Reply:
In the meanwhile, can we, for mule 3 only, add a prefix to the file name, something like mule3-<filename> or m3-<filename>. The idea would be to group somehow, all archives in the same way.

Reply:
I added -mule3 to all Mule 3 policies related articles: https://github.com/mulesoft/docs-api-manager/pull/20/commits/d3449a8fa4c34f6a8bb7276480d1f4194ebe2774

> policies-mule3.adoc
available-policies-mule3.adoc
using-policies-mule3.adoc
setting-your-api-url-mule3.adoc
reorder-policies-task-mule3.adoc
tutorial-manage-an-api-mule3.adoc
resource-level-policies-mule3.adoc
prepare-raml-mule3.adoc
disable-edit-remove-mule3.adoc
provided-policies-mule3.adoc
add-remove-headers-mule3.adoc
cors-policy-mule3.adoc
client-id-based-policies-mule3.adoc
http-basic-authentication-policy-mule3.adoc
ip-blacklist-mule3.adoc
ip-whitelist-mule3.adoc
json-threat-mule3.adoc
xml-threat-mule3.adoc
ldap-security-manager-mule3.adoc
throttling-rate-limit-mule3.adoc
rate-limiting-and-throttling-sla-based-policies-mule3.adoc
apply-rate-limiting-mule3.adoc
rate-limiting-and-throttling-mule3.adoc
aes-oauth-faq-mule3.adoc
mule-oauth-2.0-token-validation-policy-mule3.adoc
openam-oauth-token-enforcement-policy-mule3.adoc
apply-oauth-token-policy-mule3.adoc
custom-policies-mule3.adoc
creating-custom-policy-mule3.adoc
custom-policy-references-mule3.adoc
pointcut-reference-mule3.adoc
resource-level-custom-policy-mule3.adoc